dbt/loader.py

```diff
@@ -59,7 +66,8 @@ def _load_seeds(self):
                 all_projects=self.all_projects,
                 root_dir=project.project_root,
                 relative_dirs=project.data_paths,
-                macros=self.macros
+                macros=self.macros,
```

Comment:
A future cleanup PR could probably go back and remove the `macros` parameter here since they're part of the manifest, but I suspect it's actually a bit tricky to do so.